packages/connection-router-api/src/error.rs

```diff
@@ -0,0 +1,14 @@
+use thiserror::Error;
+
+/// A chain name must adhere to the following rules:
+/// 1. it can optionally start with an uppercase letter, followed by one or more lowercase letters
+/// 2. it can have an optional suffix of an optional dash and one or more digits ("1", "03", "-5" are all valid suffixes)
+pub const CHAIN_NAME_REGEX: &str = "^[A-Z]?[a-z]+(-?[0-9]+)?$";
```

Comment:
Have this unit tested, which also serves as examples for valid chain names.

Reply:
what do you mean? there are  a bunch of tests for the chain names